components/cdc/src/endpoint.rs

```diff
@@ -46,6 +46,8 @@ use crate::metrics::*;
 use crate::service::{CdcEvent, Conn, ConnID, FeatureGate};
 use crate::{CdcObserver, Error, Result};
 
+const FEATURE_RESOLVED_TS_STORE: Feature = Feature::require(5, 0, 0);
```

Comment:
CDC has its own feature gate structure, could you unify them ?

Reply:
They have different purposes, this is for checking the tikv cluster version.